runtime/v2/README.md

```diff
@@ -337,8 +347,8 @@ Messages will automatically be output in the containerd's daemon logs with the c
 
 #### ttrpc
 
-[ttrpc](https://github.com/containerd/ttrpc) is the only currently supported protocol for shims.
+[ttrpc](https://github.com/containerd/ttrpc) is one of the supported protocols for shims.
```

Comment:
nit: the phrasing here is a bit awkward

```suggestion
[ttrpc](https://github.com/containerd/ttrpc) is the recommended protocols for shims.
```

Reply:
@samuelkarp It seemed redundant to state it's recommended as there's a blurb at the bottom was my thought process 🤷‍♂️ Would you wanna change to this and get rid of the 
> It is recommended to use ttrpc in your shim but grpc support is currently an experimental feature.

Or I guess the bottom could be "As stated, it is recommended to use ttrpc in your shim but grpc support is currently an experimental feature."